CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 Full documentation for RPP is available at [https://rocm.docs.amd.com/projects/rpp/en/latest](https://rocm.docs.amd.com/projects/rpp/en/latest)
 
+## (Unreleased) RPP 1.9.5
+
+### Changes
+
+* RPP Fog augmentation on HOST and HIP
+
 ## (Unreleased) RPP 1.9.4
```

Comment:
Is 1.9.4 unreleased?

Reply:
Yes. This will go into next ROCm release